client/src/components/Search.js

```diff
@@ -1,177 +1,180 @@
-              format="MM/dd/yyyy"
-              margin="normal"
-              className={classes.input}
-              label="Departure"
-              value={state.departureDate}
-              minDate={new Date()}
-              onChange={(e) => setState({ ...state, departureDate: e })}
-            />
-            <Divider orientation="vertical" flexItem />
-          </Grid>
-          <Grid item xs={12} sm={2} className={classes.item}>
-            <FormControl className={classes.formControl}>
-              <InputLabel shrink>Travellers</InputLabel>
-              <Select
-                className={classes.input}
-                name="numOfTravellers"
-                value={state.numOfTravellers}
-                onChange={handleChange}
-              >
-                <MenuItem value={1}>1</MenuItem>
-                <MenuItem value={2}>2</MenuItem>
-                <MenuItem value={3}>3</MenuItem>
-              </Select>
-            </FormControl>
-            <Divider orientation="vertical" flexItem />
-          </Grid>
-          <Grid item xs={12} sm={2} className={classes.item}>
-            <FormControl className={classes.formControl}>
-              <Button
-                class={classes.btn}
-                variant="contained"
-                type="submit"
-                onClick={handleSubmit}
-              >
-                {" "}
-                Search
-              </Button>
-            </FormControl>
-          </Grid>
-        </Grid>
-      </MuiPickersUtilsProvider>
-    </div>
-  );
+					<Grid item xs={12} sm={2} className={classes.item}>
+						<KeyboardDatePicker
+							disableToolbar
+							variant="inline"
+							format="MM/dd/yyyy"
+							margin="normal"
+							className={classes.input}
+							label="Departure"
+							value={state.departureDate}
+							minDate={new Date()}
+							onChange={(e) =>
+								setState({ ...state, departureDate: e })
+							}
+						/>
+						<Divider orientation="vertical" flexItem />
+					</Grid>
+					<Grid item xs={12} sm={2} className={classes.item}>
+						<FormControl className={classes.formControl}>
+							<InputLabel shrink>Travellers</InputLabel>
+							<Select
+								className={classes.input}
+								name="numOfTravellers"
+								value={state.numOfTravellers}
+								onChange={handleChange}
+							>
+								<MenuItem value={1}>1</MenuItem>
+								<MenuItem value={2}>2</MenuItem>
+								<MenuItem value={3}>3</MenuItem>
+							</Select>
+						</FormControl>
+						<Divider orientation="vertical" flexItem />
+					</Grid>
+					<Grid item xs={12} sm={2} className={classes.item}>
+						<FormControl className={classes.formControl}>
+							<Button
+								class={classes.btn}
+								variant="contained"
+								type="submit"
+								onClick={handleSubmit}
+							>
+								{" "}
+								Search
+							</Button>
+						</FormControl>
+					</Grid>
+				</Grid>
+			</MuiPickersUtilsProvider>
+		</div>
+	);
```

Comment:
Do you have prettier installed? Some of this spacing seems off compared to the style the rest of the team is using